PowerEditor/src/ScintillaComponent/columnEditor.cpp

```diff
@@ -448,7 +448,7 @@ intptr_t CALLBACK ColumnEditorDlg::run_dlgProc(UINT message, WPARAM wParam, LPAR
 	return FALSE;
 }
 
-void ColumnEditorDlg::switchTo(bool toText)
+void ColumnEditorDlg::switchToText(bool toText)

```

Comment:
How about `switchToSection`?

Reply:
>How about `switchToSection`?

My goal is, as always, to change as little as possible, while still adding important value, because this lessens the chance of rejection.

It is challenging to read another developer's mind (Don's) and try to guess what is acceptable vs. not acceptable.

If Don likes something else, he'll tell me/us.

Notes:

* Changing the original `switchTo` has nothing whatsoever to do with fixing the bug; I only changed it because it suffers from the same problem as the source of the bug (poor naming).

* In my own code I would have made it `switchToTextNotNumberSection`, because that is the way I do things.

Reply:
> In my own code I would have made it switchToTextNotNumberSection

I'm not sure I understand it.
`switchTo()` switches either to Text or to Number.

Reply:
And **_I_** don't understand your followup...
But my main point was that everyone has a different way of doing things.

Reply:
OK.
Thanks again for this PR.
It fixes the bug and the code is much clearer. 👍